src/pages/VenuePage/VenuePage.tsx

```diff
@@ -132,10 +135,22 @@ export const VenuePage: React.FC = () => {
 
   // @debt refactor how user location updates works here to encapsulate in a hook or similar?
   useEffect(() => {
-    if (!userId || !venueName) return;
+    if (!userId || !venueName || !venueId) return;
+
+    const getWholePath = async () => {
+      const { sovereignVenue, checkedVenueIds } = await fetchSovereignVenue(
+        venueId
+      );
+
+      const allVenues = [...checkedVenueIds, sovereignVenue.id].reverse();
+
+      const locationPath = wrapIntoSlashes(allVenues.join("/"));
+
+      setLocationData({ userId, locationPath });
+    };
 
-    setLocationData({ userId, locationName: venueName });
-  }, [userId, venueName]);
+    getWholePath();
```

Comment:
Nitpick - This method call has a side effect, so I'd rename it so it's clear it's setting something, not getting something.
e.g. `setWhoPathLocationData`

Reply:
Renamed in https://github.com/sparkletown/sparkle/pull/2202/commits/a595fe8a8fe16360a45f0a9ee992430f5e28ac9b